src/ampJson.js

```diff
@@ -1,7 +1,14 @@
-export default function ampJson(data, component) {
+export default function ampJson(data, component, attributes={}) {
   if (!data || !component) return '';
+
+  const attributesString = attributes ?
+    Object.entries(attributes).map(([key, val]) => (
+      `${key}=${val}`
+    )).join(' ').trim() :
+    undefined;
+
   return Object.entries(data).map(([key, value]) => (
-    `<${component} id="${key}">
+    `<${component} id="${key}" ${attributesString}>
```

Comment:
I think I'm not being very clear, my apologies.

You're welcome to use whatever options schema you think makes sense - the issues is that this is generating a string of HTML, so *nothing* from the user is safe to inject into it without being sanitized. Only `value` is being sanitized here (by `JSON.stringify`).